guides/doc-Project_API/topics/proc_using-the-post-http-method.adoc

```diff
@@ -0,0 +1,74 @@
+[id="using-the-post-http-method"]
+= Using the POST HTTP method
+
+Use the POST HTTP verb to submit data to the API to create an entry or resource.
+You must submit the data in JSON format.
+For more information, see xref:sect-API_Guide-Passing_JSON_Data_with_the_API_Request[].
+
+.Example
+This example creates an activation key.
+
+. Create a test file, for example, `activation-key.json`, with the following content:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+{"organization_id":1, "name":"TestKey", "description":"Just for testing"}
+----
+
+. Create an activation key by applying the data in the `activation-key.json` file:
++
+Example request:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ curl --header "Accept:application/json" \
+--header "Content-Type:application/json" --request POST \
+--user _sat_username:sat_password_ --insecure \
+--data @activation-key.json \
+https://_{foreman-example-com}_/katello/api/activation_keys \
+| python3 -m json.tool
+----
++
+Example response:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+{
+    "id": 2,
+    "name": "TestKey",
+    "description": "Just for testing",
+    "unlimited_hosts": true,
+    "auto_attach": true,
+    "content_view_id": null,
+    "environment_id": null,
+    "usage_count": 0,
+    "user_id": 3,
+    "max_hosts": null,
+    "release_version": null,
+    "service_level": null,
+    "content_overrides": [
+
+    ],
+    "organization": {
+        "name": "Default Organization",
+        "label": "Default_Organization",
+        "id": 1
+    },
+    "created_at": "2017-02-16 12:37:47 UTC",
+    "updated_at": "2017-02-16 12:37:48 UTC",
+    "content_view": null,
+    "environment": null,
+    "products": null,
+    "host_collections": [
+
+    ],
+    "permissions": {
+        "view_activation_keys": true,
+        "edit_activation_keys": true,
+        "destroy_activation_keys": true
+    }
+}
+----
+
+. Verify that the new activation key is present.
```

Comment:
```suggestion
.Verification
* Verify that the new activation key is present.
```

I think we can also do this in a follow-up PR; I assume that lots of procedures could contain a "Verification" section at the end that checks if Foreman API accepted the user's request.